benches/lib.rs

```diff
@@ -41,6 +41,19 @@ mod tests {
       }
     });
   }
+  #[bench]
+  fn bench02_pooled_scoped_allocation_speed(b: &mut Bencher) {
```

Comment:
Maybe it would work to write the name in prefix form instead of numbering the benchmarks? Assuming they are just ordered lexically this should work:
```
  fn allocation_speed_standard(b: &mut Bencher)
  fn allocation_speed_pooled(b: &mut Bencher)
  fn allocation_speed_pooled_scoped(b: &mut Bencher)
```